components/campaign/CampaignTable.js

```diff
@@ -20,37 +24,103 @@ export default function CampaignTable (props) {
       km_coastlines_add_mod: user.km_coastlines_add + user.km_coastlines_mod,
       km_waterways_add_mod: user.km_waterways_add + user.km_waterways_mod
     }))
+
+  const headerDivs = prepareAllHeaders(tableHeaderNames.CAMPAIGN)
+  const formattedNum = ({ cell: { value } }) => formatDecimal(value)
+  const { headers, rows, prepareRow } = useTable({
+    columns: [
+      {
+        Header: headerDivs['name'],
+        accessor: 'name',
+        Cell: ({ cell: { value } }) => (
+          <Link href={`/users/${idMap[value]}`}>
+            <a className='link--normal' >
+              { value }
+            </a>
+          </Link>
+        )
+      },
+      {
+        Header: headerDivs['roads'],
+        accessor: 'km_roads_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['buildings'],
+        accessor: 'buildings_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['poi'],
+        accessor: 'poi_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['railways'],
+        accessor: 'km_coastlines_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['coastlines'],
+        accessor: 'km_coastlines_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['waterways'],
+        accessor: 'km_waterways_add_mod',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['changeset'],
+        accessor: 'changeset_count',
+        Cell: formattedNum
+      },
+      {
+        Header: headerDivs['edits'],
+        accessor: 'edit_count',
+        Cell: formattedNum
+      }
+    ],
+    data: campaignTopStats
+  },
+  useSortBy
+  )
+
   return (
     <div className='widget clearfix table-wrapper'>
       <table>
         <thead>
           <tr>
-            <TableHeaders tableName={tableHeaderNames.CAMPAIGN} />
+            {
+              headers.map(column =>
+                <th {...column.getHeaderProps(column.getSortByToggleProps())}>
+                  {column.Header}
```

Comment:
Why not use the react-table sortMethod in the column array above?

Reply:
I'm not sure what you're referring to?